ipfinder/metadata/ip_finder_metadata.go

```diff
@@ -0,0 +1,51 @@
+package metadata
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancher/go-rancher-metadata/metadata"
+	"time"
+)
+
+const (
+	metadataURL = "http://rancher-metadata/latest"
```

Comment:
please point to the specific version of metadata. Using "latest" can bite you later